src/Hl7.Fhir.Shared.Core/Model/ModelInfo.cs

```diff
@@ -310,7 +310,7 @@ public static ModelInspector ModelInspector
             typeof(Model.Distance),
             typeof(Model.Dosage),
             typeof(Model.Duration),
-            typeof(Model.Expression),
+          //  typeof(Model.Expression),
```

Comment:
Why is this removed?

Reply:
Oops, well spotted.

Reply:
Solved: `ModelInfo.OpenTypes` has been moved to the satellites.